hack/update-payload-crds.sh

```diff
@@ -4,7 +4,6 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 crd_globs="\
     authorization/v1/zz_generated.crd-manifests/*_config-operator_*.crd*yaml\
-    config/v1/zz_generated.crd-manifests/*_config-operator_*.crd*yaml\
```

Comment:
Why is this change required?

Reply:
Add this change so the v1 manifests will not be included in the payload at this time https://github.com/openshift/api/tree/master/payload-manifests/crds. From our previous [discussion](https://redhat-internal.slack.com/archives/CE4L0F143/p1747639470044449?thread_ts=1747575988.901259&cid=CE4L0F143), we will include the manifests later when MCO controller implementation is ready to use v1 APIs.